storage/src/adb/current.rs

```diff
@@ -102,70 +102,82 @@ impl<
         };
 
         let context = context.with_label("adb::current");
-        let mut status = Bitmap::<H, N>::restore_pruned(
+        let mut status = Bitmap::restore_pruned(
             context.with_label("bitmap"),
             &config.bitmap_metadata_partition,
         )
         .await?;
 
         // Initialize the db's mmr/log.
-        let (mmr, log) =
-            Any::<_, _, _, _, T>::init_mmr_and_log(context.clone(), hasher, cfg).await?;
+        let mut hasher = H::new();
+        let mut b_hasher = Basic::new(&mut hasher);
+        let (mut mmr, log) =
+            Any::<_, _, _, _, T>::init_mmr_and_log(context.clone(), &mut b_hasher, cfg).await?;
```

Comment:
In both files, I believe this function takes a `Basic` hasher. Should it just take a reference to `H` instead?

Reply:
Good catch, changed.